newsfragments/676.bugfix.rst

```diff
@@ -0,0 +1 @@
+Destructor methods (``__del__``) are now protected against ``KeyboardInterrupt``
```

Comment:
Nit: can you add a period to the end here, to match the style of the other newsfragments?